.github/workflows/production.yml

```diff
@@ -31,14 +31,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout project
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Use Node.js 16.xs
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
```

Comment:
This might be trickier :cry: :thinking: 

See here (and similar places) the action as precisely uses `node-version: '16.x'`

Reply:
I don't know why it's setup this way, but I've been running k6-docs locally with node v20, so maybe we can just update any ` node-version` lines to 20?

Reply:
Can we remove them instead ?

Reply:
I think we need to keep these actions in case we need to make any updates to k6.io/docs, at least until the website is live.

Reply:
If we are going to be dropping it - let's not try to fix it perfectly now.